draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -259,6 +259,9 @@ Additionally, there are several terms used throughout this document to describe
 Assertion:
 : A protocol-specific statement that the CA is certifying. For example, in TLS, the assertion is that a TLS signing key can speak on behalf of some DNS name or other identity.
 
+Abridged assertion:
+: A partially-hashed *Assertion* to save space. For example, in TLS, an abridged assertion replaces the subject public key by a hash.
```

Comment:
> Re "For example, ...", this isn't really a TLS-specific construction, right? We're just hashing the subject naively.

At the moment we are, but we could vary that for other subject types. Reason I add TLS explicitly as an example, is that it is easier to follow: the reader might not know what *subject information* is.

> we haven't really been italicizing structure names

Removed italicisation. It was meant more as emphasis instead of consistent style.